src/http_seed_connection.cpp

```diff
@@ -218,7 +218,7 @@ namespace libtorrent
 
 		if (error)
 		{
-			received_bytes(0, bytes_transferred);
+			received_bytes(0, int(bytes_transferred));
```

Comment:
maybe received_bytes() should really take a size_t as well. it seems like that may also cut down on the casting, what do you think?